internal/pkg/cli/deploy/env.go

```diff
@@ -156,9 +163,11 @@ func NewEnvDeployer(in *NewEnvDeployerInput) (*envDeployer, error) {
 			TemplatePatcher: cfnClient,
 			Env:             in.Env,
 		},
-		newStackSerializer: func(in *cfnstack.EnvConfig, lastForceUpdateID string, oldParams []*awscfn.Parameter) stackSerializer {
-			return cfnstack.NewEnvConfigFromExistingStack(in, lastForceUpdateID, oldParams)
+		newStack: func(in *cfnstack.EnvConfig, lastForceUpdateID string, oldParams []*awscfn.Parameter) deploycfn.StackConfiguration {
+			conf := cfnstack.NewEnvConfigFromExistingStack(in, lastForceUpdateID, oldParams)
+			return deploycfn.WrapWithTemplateOverrider(conf, overrider)
 		},
+		overrider:    overrider,
```

Comment:
Just a quick question, do we need overrider separately here? Or we can just use it from the `newStack`?
Otherwise looks good to me.

Reply:
😍 thanks for pointing this out! I added this field earlier before the `newStack` refactor and forgot to remove 🤦 Fixed!